sessionspaces/src/resources/config_maps.rs

```diff
@@ -0,0 +1,64 @@
+use k8s_openapi::api::core::v1::ConfigMap;
+use kube::{
+    api::{ObjectMeta, Patch, PatchParams},
+    Api, Client,
+};
+use std::collections::{BTreeMap, BTreeSet};
+use tracing::{info, instrument};
+
+const POLICY_CONFIG: &str = "policy-config";
```

Comment:
Would be good to have it named something Sessionspace related, maybe even just `sessionspace`